### DISCUSSION TITLE
[@types/dompurify] Missing ALLOW_ARIA_ATTR configuration

### DISCUSSION THREAD

**Discussion by lnewson** [DefinitelyTyped/DefinitelyTyped#58135] [Issues with a @types package] [answered]
As per https://github.com/cure53/DOMPurify/blob/45ee75572a9e1c04b031133796eda49a516c81bb/src/purify.js#L410, DOMPurify allows ARIA attributes to be allowed/blocked using the `ALLOW_ARIA_ATTR` configuration. However, this property is currently missing in the `Config` type provided.

Edit: Have logged https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58136 to fix this.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
closed in  #58136
